packaging/installer/methods/macos.md

```diff
@@ -36,6 +37,25 @@ brew install netdata
 
 Skip on ahead to the [What's next?](#whats-next) section to find links to helpful post-installation guides.
 
+## Install Netdata with MacPorts
+
+To install Netdata with MacPorts, simply run:
+
+```bash
+sudo port install netdata
+```
+
+You can have MacPorts automatically run the `netdata` service in the background and on restarts with:
+
+```bash
+sudo port load netdata
+```
+
+Your Netdata config file will be placed at `/opt/local/etc/netdata/netdata.conf`. You can find stock configuration
+files at `/opt/local/lib/netdata/conf.d/`.
```

Comment:
Is this path valid for m1 devices?

Reply:
Yes, MacPorts uses the same prefix on all devices.